freeplane/src/external/resources/xml/preferences.xml

```diff
@@ -166,6 +166,7 @@
 				<combo name="standardcloudshape" enum="org.freeplane.features.cloud.CloudShape"/>
 			</separator>
 			<separator name="connectors">
+				<boolean name="assignsNodeDependantStylesToNewConnectors"/>
```

Comment:
`assigns` (third person) instead of `assign` - is this intentional?
I can see the other property names use the imperative form rather than third-person

Reply:
Now I intentionally use the third person form for boolean properties.
It might deviate from earlier named properties.